.env

```diff
@@ -5,3 +5,5 @@ PORT=8000
 
 JWT_SECRET=@McQfTjWnZr4u7x!A%D*F-JaNdRgUkXp
 TOKEN_DURATION=30d
+
+MAX_PATIENTS_FOR_NURSE=6
```

Comment:
Remove this file and add a `.env.example` instead.
The content of the `.env.example` should be like this: 

```
JWT_SECRET=CHANGE_ME
TOKEN_DURATION=CHANGE_ME
MAX_PATIENTS_FOR_NURSE=CHANGE_ME
```

Reply:
Also remove this file from the repo.